applications/crossbar/src/modules/cb_port_requests.erl

```diff
@@ -415,19 +415,8 @@ post(Context, Id) ->
     do_post(Context, Id).
 
 post(Context, Id, ?PORT_SUBMITTED) ->
-    _  = add_to_phone_numbers_doc(Context),
-    try send_port_request_notification(Context, Id) of
-        _ ->
-            lager:debug("port request notification sent"),
-            do_post(Context, Id)
-    catch
-        _E:_R ->
-            lager:debug("failed to send the port request notification: ~s:~p", [_E, _R]),
-            cb_context:add_system_error('bad_gateway'
-                                        ,<<"failed to send port request email to system admins">>
-                                        ,Context
-                                       )
-    end;
+    DryRun = (not wh_json:is_true(<<"accept_charges">>, cb_context:req_json(Context), 'false')),
```

Comment:
I'd use `wh_util:is_true(cb_context:req_value(Context, <<"accept_charges">>, 'false'))` instead, since you could specify "accept_charges" on the query string.

We should probably put this in a function in cb_context too: `cb_context:accepting_charges(Context, [Default])`